components/automate-chef-io/content/docs/backup.md

```diff
@@ -129,6 +129,15 @@ The command will show the backup progress for each service. If the backup is suc
 Success: Created backup 20180518010336
 ```
 
+## Storing Backups in a Single-file Archive for Airgapped Installations
+
+The backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory. 
+You will need to archive both of these directory types in any offline backups you create.
```

Comment:
I translated the original "these" in this sentence as "directory types" but please feel free to amend if that's incorrect!